testsuite/ui/views/admin/audience/billing.py

```diff
@@ -24,8 +24,14 @@ class ChargingForm(View):
     """Charging form for 3scale billing"""
     ROOT = '//form[contains(@id, "edit_finance_billing_strategy")]'
     charging = ThreescaleCheckBox('//input[@id="finance_billing_strategy_charging_enabled"]')
+    currency = ThreescaleDropdown("//*[@id='finance_billing_strategy_currency']")
     save_btn = ThreescaleSubmitButton()
 
+    def change_currency(self, value):
+        """Change 3scale billing currency"""
+        self.currency.select_by_value(value)
```

Comment:
This action will be typically needed only once per 3scale deployment. I'd like to see if a statement verifies if the currency is set.